scripts/create-kubestellar-demo-env.sh

```diff
@@ -103,27 +103,42 @@ cluster_clean_up "kind delete cluster --name kubeflex" &
 cluster_clean_up "kind delete cluster --name cluster1" &
 cluster_clean_up "kind delete cluster --name cluster2" &
 wait
-echo -e "Cluster space clean up has been completed"
+echo -e "\033[33m✔\033[0m Cluster space clean up has been completed"
 
 echo -e "\nStarting context clean up..."
 context_clean_up
-echo "Context space clean up completed"
+echo -e "\033[33m✔\033[0m Context space clean up completed"
 
 echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
 clusters=(cluster1 cluster2)
 for cluster in "${clusters[@]}"; do
    (
      echo -e "Creating cluster ${cluster}..."
      kind create cluster --name "${cluster}" >/dev/null 2>&1 &&
-     kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
-     echo -e "${cluster} creation and context setup complete"
+     echo -e "\033[33m✔\033[0m ${cluster} creation and context setup complete"
    ) &
 done
 wait 
 
+for cluster in "${clusters[@]}"; do
+   kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
+done
+
+for cluster in "${clusters[@]}"; do
+  if kubectl config get-contexts | grep -w "\b${cluster}\b" >/dev/null 2>&1; then
```

Comment:
Doesn't the change, above, to doing the context renamings serially remove the need to check and retry?